static/app/components/replays/replayContext.tsx

```diff
@@ -629,7 +642,7 @@ function ProviderNonMemo({
         fastForwardSpeed,
         addHighlight,
         initRoot,
-        isBuffering,
+        isBuffering: isBuffering && !isVideoReplay,
```

Comment:
I noticed this bug where the old isBuffering would trigger whenever we would seek, so this is only for non-video replays now.